kevin-charpentier-southsaintpaul.md

```diff
@@ -0,0 +1,10 @@
+# South Saint Paul, MN
+
+### Restaraunts to Visit
+ - Black Sheep
+
+### Natural Places
+ - 
+
+### Stores
+ - Level Up
```

Comment:
@kevin-charpentier I would suggest adding the store at which you service your motorbike and they call you kim.